doc/UML/user_and_device_authentication.uml

```diff
@@ -0,0 +1,24 @@
+@startuml
+
+title A known device login on a Caliopen instance
+
+== Authentication ==
+User -> Device: give username and password
+Device -> Device: private_key = AES.decrypt(crypted_key, password)
+Device -> Device: sign_auth = private_key.sign(password)
+
+Device -> API: POST /authentications (username, password, sign_auth)
+API -> Backend: check_credentials
+Backend -> API: access token
+API -> Backend: check_device_signature(device_id, sign(device_id))
+Backend -> API: ok
+API -> Device: 200
+
+== Call API ==
+Device -> Device: sign_call = sign(access_token)
+Device -> API: GET /discussions with headers Access-Token, X-Device-ID X-Device-Signature
```

Comment:
Yes, in `Authorization` header (https://github.com/CaliOpen/caliopen.web/blob/master/app/routes/api.js#L18)